crates/cargo-bsp/src/project_model/project_manifest.rs

```diff
@@ -1,3 +1,5 @@
+//! Discovers Cargo project manifest.
```

Comment:
Cargo project's manifest discovery, in order to call Cargo metadata properly, up2u